Pearson-Advance/openedx-themes
# PAE-173: Update LBS background image

## Description:

This PR replaces the LBS theme homepage background image with a new image.

## Screenshots:

![Homepage Screenshot (Desktop)](https://user-images.githubusercontent.com/8495728/184716040-a9e45b51-eaed-4f06-baaf-c73980fa90d4.png)
![Homepage Screenshot (XXL)](https://user-images.githubusercontent.com/8495728/184952076-dfac540b-5aba-4fbf-8ba1-a07c6a8800b1.png)
![Homepage Screenshot (Mobile)](https://user-images.githubusercontent.com/8495728/184716162-eeaa6ddd-6e88-4a63-a782-ea3926cd2d40.png)

## Testing:

- Check out this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets, paver update_assets --themes=pearson-lbs-theme
- Add 'css_overrides_file' setting to a new site.
- Assign 'pearson-lbs-theme' to the newly created site.

## Reviewers:
